tesseract/tesseract_planning/tesseract_process_managers/include/tesseract_process_managers/process_input.h

```diff
@@ -105,11 +105,19 @@ struct ProcessInput
   /** @brief Results/Seed for this process */
   Instruction* results;
 
+  void setStartInstruction(Instruction instruction);
```

Comment:
Should there be a setStartInstructionPtr(Instruction* instruction)?

Reply:
After going through the rest of the changes, this function will need to be available so you can reference composite instruction in the final results data structure which get populated by preceding tasks.